### PR TITLE
Added Closing Time Keys Upgrade

### PR DESCRIPTION
**Pull request by BowlsforRick**

### CODE REVIEW

**Comment by LinosM** [comment]
- You have an extra Track name event in both PART KEYS and VENUE
track name1
track name2

- Double check REAL_KEYS_E and REAL_KEYS_M, for some reason you have notes that are different from expert. Most notably from measure 74 and beyond
different notes

- I would add an extra OD marker on 45
- Pull back the trainer markers 89/93 a measure so they land on 88/92
- The sustain on PK_E on 77.3 ends prematurely compared to the other diffs
- Add `(solo (keys))` to the .dta

**Comment by BowlsforRick** [comment]
Must've forgot to run the midi through magma, I made all the changes above and found some bad midi notes to remove

**Comment by LinosM** [comment]
Hard 5L/PK is reduced in a weird way, everything from 5-48, 63-66, 88-105: restore the note on X.3.50 and delete the note on X.4.50 instead.

Restore the notes on 51.2.00 & 61.2.00 on both Hard and Medium, along with 62.4.00 on Medium

Delete notes 57.4.00, 59.1.50, 59.4.00, 61.1.50, 61.4.00, & 62.3.50 on Medium 5L/PK

Easy 5L move the Y's on 58.2.50, 60.2.50, & 62.2.50 up to B

Medium PK move the G# on 89.4.00 down to G

**Comment by BowlsforRick** [comment]
Reduction changes have been made to mid. file as per recommendations

File: Pro Keys (No New Audio)/Rock Band 4 DLC/closingtime/songs.dta [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+   (
+      'album_name'
+      "Feeling Strangely Fine"
+   )
+   ('album_track_number' 1)
+   ('context' 2000)
+   ('decade' 'the90s')
+   (downloaded TRUE)
+   ('base_points' 0)
+   (vocal_tonic_note 4)
```
Comment (LinosM) [inline Pro Keys (No New Audio)/Rock Band 4 DLC/closingtime/songs.dta:111]:
```suggestion
   (vocal_tonic_note 7)
```
Song is in G major

midi file itself looks good now

**Pull request merged**